test/Exercism.Representers.CSharp.IntegrationTests/Solutions/ReadOnly/Fields/Fake.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+public class Fake
+{
+    private readonly int roField = 1729; 
```

Comment:
Maybe also add a static field? It should not matter, but just for the test coverage.

Reply:
added